assertk-jvm/src/main/kotlin/assertk/assertions/any.kt

```diff
@@ -1,80 +1,17 @@
+@file:JvmName("AnyJVMKt")
```

Comment:
This is a breaking change for java users, but i don't think it'll be a problem since `assertk` isn't really used from `java`.

Reply:
Not to worried about the break here, though just curious why this is required?

Reply:
It's required because the file already exists in the common module. So there is a conflict between common code and java specific code.
Another solution that I like is to create another file name `JVMExtensions` or something like that.